### ISSUE TITLE
Support for @ mentions auto-complete

### ISSUE THREAD

**Issue by bjubb** [dennisreimann/ioctocat#227]
When commenting on a pull req for example, when I type an @, I'd like auto complete to start with a filtered list as I type the name or nickname of the user I'm directing a specific comment towards.   This is something the GitHub site can do but I have no idea if their API provides an endpoint for retrieving users per organization.   Anyways-would be quite helpful as one name typo for my workflow would not produce a proper push  notification to that user.

**Comment by dennisreimann** [comment]
Nice idea. There is no endpoint for something like that besides the user search, but I can imagine that we could use our internal user cache for that. It's likely that someone you'll want to mention is in there, because it contains every user the app has come across.

**Comment by bjubb** [comment]
I's there any Org info that comes along with user data in the cache for
limiting results based upon repos ?

Sent from my iPhone

On Jan 29, 2013, at 8:32 AM, Dennis Reimann notifications@github.com
wrote:

Nice idea. There is no endpoint for something like that besides the user
search, but I can imagine that we could use our internal user cache for
that. It's likely that someone you'll want to mention is in there, because
it contains every user the app has come across.

—
Reply to this email directly or view it on
GitHubhttps://github.com/dennisreimann/ioctocat/issues/227#issuecomment-12834447.

**Comment by dennisreimann** [comment]
I'll figure something out for that. Don't expect this feature to come very soon, because I got some more things to work on, but I added it to the 2.0 milestone. Thanks for bringing it up, I like the idea :)

**Comment by bjubb** [comment]
No problem.   App works great for how Managing my non- coding duties!

Sent from my iPhone

On Jan 29, 2013, at 11:12 AM, Dennis Reimann notifications@github.com
wrote:

I'll figure something out for that. Don't expect this feature to come very
soon, because I got some more things to work on, but I added it to the 2.0
milestone. Thanks for bringing it up, I like the idea :)

—
Reply to this email directly or view it on
GitHubhttps://github.com/dennisreimann/ioctocat/issues/227#issuecomment-12842254.
